### ISSUE TITLE
Can't open remote folder from an azure repos

### ISSUE THREAD

**Issue by aline-siqueira-ey** [microsoft/vscode-tools-for-ai#2035]
Does this occur consistently? 
Repro steps:

1.
2.

Action: Resolver.resolve
Error type: 70
Error Message: request to redacted:url failed, reason: unable to get local issuer certificate

Version: 0.29.2023050909
OS: win32
OS Release: 10.0.19044
Product: Visual Studio Code
Product Version: 1.78.1
Language: en

<details>
<summary>Call Stack</summary>

```
s extension.js:2:1984620
extension.js:2:2010345extension.js:2:2010345
```

</details>

**Comment by sevillal** [comment]
Hi @aline-siqueira-ey, I apologize for the issue you are facing.

- Could you please try installing this extension to diagnose problems like this: https://marketplace.visualstudio.com/items?itemName=chrmarti.network-proxy-test. 
- Then use F1 > Network Proxy Test: Test Network Connection with the URL is failing (I assume it might be something like https://..instances.azureml.ms/api/terminals) to check if that works. 
- If it shows the certificate error retry with F1 > Network Proxy Test: Test Network Connection (Allow Unauthorized) to get more information about the problem.
- Also run F1 > Network Proxy Test: Show OS Certificates and share it's outpu

Make sure replace any sensitive information before posting it here. Thanks.

**Comment by sevillal** [comment]
@aline-siqueira-ey VS Code just released version 1.78.2 which should fix the certificates issue, could you please update VS Code (Help > Check for Updates...) and retry the connection.
